src/engine/qcommon/q_math.cpp

```diff
@@ -36,6 +36,9 @@
 
 #include "q_shared.h"
 
+#define DotProduct4(x, y)            (( x )[ 0 ] * ( y )[ 0 ] + ( x )[ 1 ] * ( y )[ 1 ] + ( x )[ 2 ] * ( y )[ 2 ] + ( x )[ 3 ] * ( y )[ 3 ] )
+#define VectorMultiply( a,b,c )      ( ( c )[ 0 ] = ( a )[ 0 ] * ( b )[ 0 ],( c )[ 1 ] = ( a )[ 1 ] * ( b )[ 1 ],( c )[ 2 ] = ( a )[ 2 ] * ( b )[ 2 ] )
+
```

Comment:
why not these two?

Reply:
Since they were only used in one file, it seemed to me they are not suitable for inclusion in the header included by all files. And with only 1-2 uses in 1 file they should probably be normal functions not templates, but I didn't want to spend time on them.